src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,390 @@
+---
+title: Organize CockroachDB Cloud Clusters Using Folders
+summary: Learn how to use folders to organize CockroachDB Cloud clusters and manage access to them.
+toc: true
+docs_area: manage
+---
+
+{% include_cached feature-phases/limited-access.md %}
+
+This page explains how to organize and manage access to your {{ site.data.products.db }} organization's clusters using folders. For more details about managing access to {{ site.data.products.db }} resources, refer to [Managing Users, Roles, and Service Accounts in {{ site.data.products.db }}](managing-access.html).
+
+## How folders work
+
+Folders provide additional ways to organize clusters and manage access to them. A folder can contain a mix of folders and clusters. Roles assigned on a folder are inherited by its descendants.
+
+Each folder that you create has an ID. To create a cluster in a folder or to move a cluster into a folder, you set its `parent_id` field to the folder's ID.
+
+If no parent ID is specified for a cluster, it is created at the level of the organization. Clusters and folders at the organization level have their `parent_id` set to `root`.
```

Comment:
```suggestion
If no parent ID is specified for a cluster, it is created at the root level of the organization. Clusters and folders at the organization level have their `parent_id` set to `root`.
```

nit. Root level? Base level? Top level? I feel like this needs a little clarification, but feel free to disregard.

Reply:
It is called `root` but it is also at the root of the organization, if you consider folders to be branches. I think it's OK with your suggestion.